linux_os/guide/system/accounts/enable_authselect/ansible/shared.yml

```diff
@@ -0,0 +1,33 @@
+# platform = multi_platform_all
+
```

Comment:
you will need to add all of the metadata stuff here... disruption, severity ,etc...